ultralytics/engine/predictor.py

```diff
@@ -392,6 +392,7 @@ def show(self, p=""):
             cv2.resizeWindow(p, im.shape[1], im.shape[0])  # (width, height)
         cv2.imshow(p, im)
         cv2.waitKey(300 if self.dataset.mode == "image" else 1)  # 1 millisecond
+        cv2.destroyWindow(p) if self.dataset.mode == "image" else None
```

Comment:
Nice, but why not do it like this (easier to read):

```suggestion
if self.dataset.mode == "image":
    cv2.destroyWindow(p)
```

Reply:
Yes definitely we can do this way as well 
I just followed what was done earlier

Thank you for your comment